airflow/utils/db.py

```diff
@@ -540,6 +537,11 @@ def resetdb():
     """
     Clear out the database
     """
+    from airflow.models.all_models import (  # noqa: F401 # pylint: disable=unused-import
+        DAG, XCOM_RETURN_KEY, BaseOperator, BaseOperatorLink, BaseJob, Connection, DagBag,
+        DagModel, DagPickle, DagRun, DagTag, Log, Pool, SerializedDagModel, SkipMixin, SlaMiss,
+        TaskFail, TaskInstance, TaskReschedule, Variable, XCom
+    )
```

Comment:
@potiuk I would have placed `from airflow.models.all_models import *` as a local import as you specified, however, `import *` is only allowed at the module level in python. Therefore, I had to specify all model names individually.

I could wrap this into some kind of `load_models` helper function so it's a bit cleaner

Reply:
Yes. I think it would be nicer to do it this way indeed.